export.go

```diff
@@ -77,7 +78,12 @@ func exportPostsZip(u *User, posts *[]PublicPost) []byte {
 		if err != nil {
 			log.Error("export zip header: %v", err)
 		}
-		_, err = f.Write([]byte(file.Body))
+		var fullPost string
+		if file.Title != "" {
+			fullPost = "# " + file.Title + "\n\n"
```

Comment:
if the post has a title, will the header not already be at the top of the post?

Reply:
oh I see, so somewhere I missed the header is stripped out when creating a txt file from a post?

Reply:
The title is a separate database field, so we need to manually include it like this.

Reply:
:+1: